admin_organization.go

```diff
@@ -101,7 +101,7 @@ type AdminOrganizationID struct {
 }
 
 // List all the organizations visible to the current user.
-func (s *adminOrganizations) List(ctx context.Context, options AdminOrganizationListOptions) (*AdminOrganizationList, error) {
+func (s *adminOrganizations) List(ctx context.Context, options *AdminOrganizationListOptions) (*AdminOrganizationList, error) {
 	url := "admin/organizations"
 	req, err := s.client.newRequest("GET", url, &options)
```

Comment:
Right, so this is the only thing I think is standing out to me as an issue: 

```suggestion
	req, err := s.client.newRequest("GET", url, options)
```

In the GET functions that were already receiving `options *SomethingOptions` before this PR, it looks like we were just passing the pointer onwards to the client... but in the updated functions, we're now referencing the pointer again and passing a pointer to a pointer. 

I ran some of the updated functions in a test project (passing both `nil` and a proper pointer to an Options), and they work just fine... but for the sake of clarity and consistency, it feels like we should update the spots where we pass the options onward to use single pointers instead of nested ones.

Reply:
Indeed! When we call client.newRequest, there is no need to pass the argument options with a reference to a pointer because client.newRequest ends up receiving the address itself, rather than the value. I was trying to think if there would be a good reason to pass a pointer by address rather than passing a pointer by value but I couldn't think of anything. At least on the context of client.newRequest.
To clarify why, when you were testing, both implementation worked just fine:   

https://github.com/hashicorp/go-tfe/blob/3ea3b96bc9080a229116ba94a9f60e2bb7c10918/tfe.go#L475

the .Values function uses the Go Reflect library to unwrap the content of the interface we are passing (in this case &options). Once the Reflect library finds that it's dealing with pointers, it will dig into them until it determines if the values passed are really nil. So, even with our awful nested pointer, the .Values function did a good job in finding the value behind that pointer variable. 

Thanks for pointing this out! I have made the necessary changes. @nfagerlund